rsts/concepts/basics.rst

```diff
@@ -13,8 +13,8 @@ Concepts
        :text: Tasks
        :classes: btn-block stretched-link
     ^^^^^^^^^^^^
-    A **Task** is any independent unit of processing. They can be pure functions or functions with side-effects.
-    Each definition of a task also has associated configuration and requirements specifications.
+    A **Task** is any independent unit of processing. Tasks can be pure functions or functions with side-effects.
+    Each definition of a task also has associated configurations and requirements specifications.
```

Comment:
```suggestion
    Each task is also associated configurations and requirements.
```

Reply:
Thanks Niels. Will run the editorial review after earlier conflicts have been resolved.